tests/monotouch-test/ImageIO/CGImageAnimationTest.cs

```diff
@@ -0,0 +1,186 @@
+﻿//
+// Unit tests for CGImageAnimation
+//
+// Authors:
+//	Whitney Schmidt <whschm@microsoft.com>
+//
+// Copyright 2020 Microsoft Corp. All rights reserved.
+//
+using System;
+using System.IO;
+#if XAMCORE_2_0
+using Foundation;
+using System.Threading.Tasks;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using ImageIO;
+#else
+using MonoTouch.CoreGraphics;
+using MonoTouch.Foundation;
+using MonoTouch.ImageIO;
+using MonoTouch.UIKit;
+#endif
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.ImageIO {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class CGImageAnimationTest {
+
+		NSUrl imageUrl;
+		NSData imageData;
+		int testValue;
+		CGImageAnimation imageAnimation;
```

Comment:
Looks like we are not calling Dispose on this object, would be nice to clean after the tests in the TearDown.